Sub/Src/HSV Filtering/prominent_color.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
All the HSV stuff should be in a single script with a class like structure. I needs modularity and documentation.